packer/gpu/scripts/install-nvidia-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu -o pipefail
+
+# nvidia driver
+sudo yum install -y kernel-devel-$(uname -r) kernel-headers-$(uname -r) dkms
+
+sudo yum install -y pkgconfig libglvnd-devel
+
+DRIVER_CUDA_124="https://us.download.nvidia.com/tesla/550.54.15/NVIDIA-Linux-x86_64-550.54.15.run"
+
+curl -Lsf -o cuda_toolkit.run "${DRIVER_CUDA_124}"
+sudo dnf install -y kernel-modules-extra
+sudo sh cuda_toolkit.run --silent --dkms
+rm cuda_toolkit.run
+
+# nvidia container toolkit
+curl -s -L https://nvidia.github.io/libnvidia-container/stable/rpm/nvidia-container-toolkit.repo | sudo tee /etc/yum.repos.d/nvidia-container-toolkit.repo
```

Comment:
`curl -sfL` please. otherwise it might not fail properly when the Internet is choking.